cdk_emqx_cluster/cdk_emqx_cluster_stack.py

```diff
@@ -851,6 +885,44 @@ def setup_kafka(self):
                                      client_broker=msk.ClientBrokerEncryption.TLS_PLAINTEXT)
                                  )
 
+        if True:
```

Comment:
I haven't think about how to manage these FIS exp.  template that relies on Kafka broker's addr.

Reply:
Perhaps we could make the broker addresses an output of this stack, and then [import](https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.core/Fn.html#aws_cdk.core.Fn.import_value) that output as in input to the Chaos app?

Something like: https://bobbyhadz.com/blog/import-value-aws-cdk-cross-stack